.github/workflows/ci.yaml

```diff
@@ -20,9 +20,9 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
-          - "3.9"
           - "3.10"
-          - "3.11"
+          - "3.10"
+          - "3.12"
```

Comment:
python 3.8 and 3.9 [are still not eol](https://devguide.python.org/versions/). I know about some of the niceties that newer versions add, but for adoption of this package it might be a tad restrictive.

Reply:
that's a good point -- I was going by the [numpy schedule](https://numpy.org/neps/nep-0029-deprecation_policy.html) here, according to which python 3.9 will not be supported after april

Reply:
I will try testing on 3.8 and 3.9 and see what breaks

Reply:
Ah, I always forget about the numpy schedule... Honestly, out of practicality and the intended user-base (bio-image community, at least that's what I think) for me it's more important what [`conda-forge` currently supports](https://github.com/conda-forge/conda-forge-pinning-feedstock/blob/595a16bb6088c5c90fc7140d15ba5b507c262245/recipe/conda_build_config.yaml#L763). But I would be the first to admit that I'm biased there.